### PR TITLE
Login

### PR DESCRIPTION
**Pull request by Jayjerome**
There are some new changes in index file

### CODE REVIEW

**Comment by kevoese** [comment]
Please create a login page.

**Comment by Jayjerome** [comment]
Login page completed

File: login.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+.login {
+  max-width: 1000px;
+  min-width: 800px;
+  width: 70%;
+  height: 500px;
+  border-radius: 16px;
+  background-image: url("images/bg.png");
+  display: flex;
+  box-shadow: 1px 1px 20px 2px rgba(0, 0, 0, 0.3);
```
Comment (kevoese) [inline login.css:49]: Great job. 
include this style in the login class. it is going to fill up the page with that background
    background-size: cover;
    min-width: 900px;
height: 600px

**Review by kevoese — CHANGES_REQUESTED** [review]

File: login.css (hunk 1 above)
Comment (Jayjerome) [reply login.css:49]: Done

**Review by Jayjerome — COMMENTED** [review]
All changes effected

**Review by kevoese — APPROVED** [review]

**Pull request merged**